wordpress-mysql-replication/azuredeploy.json

```diff
@@ -237,12 +231,12 @@
     },
     "variables": {
         "template": {
-            "base": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/wordpress-mysql-replication"
+            "base": "https://raw.githubusercontent.com/Sunbuild/azure-quickstart-templates/master/wordpress-mysql-replication"
```

Comment:
I mentioned in the comment above. Here it is again : the base URL is still using Sunbuild repo link because the shell script was modified. If Azure repo link the deployment test with travisCI will fail. Once merged I will send another PR to update the baseUrl to Azure repo link